app/src/main/java/com/example/unsplashphotos/ui/ShareUtils.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.unsplashphotos.ui
+
+import android.content.Context
+import android.content.Intent
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.net.Uri
+import androidx.core.content.FileProvider.getUriForFile
+import com.example.unsplashphotos.R
+import java.io.File
+import java.io.FileOutputStream
+
+
+class ShareUtils() {
```

Comment:
Can be written using Kotlin's object declaration, you don't need to initialize ShareUtils() by hand. 
Benefits? Singleton by design. Thread-safe. Lazily initialized.
https://kotlinlang.org/docs/object-declarations.html#object-declarations-overview